docs/api/en/loaders/Loader.html

```diff
@@ -57,6 +57,18 @@ <h3>[method:void load]()</h3>
 			This method needs to be implement by all concrete loaders. It holds the logic for loading the asset from the backend.
 		</p>
 
+		<h3>[method:Promise loadAsync]( [param:String url], [param:Function onProgress] )</h3>
+		<p>
+		[page:String url] — A string containing the path/URL of the <em>.gltf</em> or <em>.glb</em> file.<br />
```

Comment:
Is this method specific to `gltf/glb` files?

Reply:
Good catch!

Reply:
Oops, good catch. Fixed in #19179